docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_prefect.md

```diff
@@ -8,258 +8,218 @@ import InProgress from '/docs/components/warnings/_in_progress.md'
+[Prefect](https://prefect.io/) is a workflow orchestration and observation platform that enables data engineers, ML engineers, and data scientists to stop wondering about their workflows. [The Prefect open source library](https://docs.prefect.io) allows users to create workflows using Python and add retries, logging, caching, scheduling, failure notifications, and much more. [Prefect Cloud](https://www.prefect.io/cloud/) offers all that goodness plus a hosted platform, automations, and enterprise features for users who need them. Prefect Cloud provides free and paid tiers.
 
-Great Expectations validations can be used to validate data passed between tasks in your Prefect flow. By validating your data before operating on it, you can quickly find issues with your data with less debugging. Prefect makes it easy to combine Great Expectations with other services in your data stack and orchestrate them all in a predictable manner.
+Prefect can be used with Great Expectations validations so that you can be confident about the state of your data. With a Prefect [deployment](https://docs.prefect.io/latest/concepts/deployments/), you can productionize your workflow and run data quality checks in reaction to the arrival of new data or on a schedule. 
 
-## The `RunGreatExpectationsValidation` task
+## Doing it
 
-With Prefect, you define your workflows with [tasks](https://docs.prefect.io/core/concepts/tasks.html) and [flows](https://docs.prefect.io/core/concepts/flows.html). A `Task` represents a discrete action in a Prefect workflow. A `Flow` is a container for `Tasks`. It represents an entire workflow or application by describing the dependencies between tasks. Prefect offers a suite of over 180 pre-built tasks in the [Prefect Task Library](https://docs.prefect.io/core/task_library/overview.html). The [`RunGreatExpectationsValidation`](https://docs.prefect.io/api/latest/tasks/great_expectations.html) task is one of these pre-built tasks. With the `RunGreatExpectationsValidation` task you can run validations for an existing Great Expectations project.
+### Install 
 
-To use the `RunGreatExpectationsValidation`, you need to install Prefect with the `ge` extra:
+Install the Great Expectations, Prefect, and [prefect-great-expectations](https://prefecthq.github.io/prefect-great-expectations/) libraries into the same Python virtual environment. 
 
 ```bash
-pip install "prefect[ge]"
+pip install great_expectations==0.16.12 prefect==2.10.10 prefect_great_expectations==0.1.0
```

Comment:
Are the version pins necessary for these 3 packages?

Reply:
```suggestion
pip install great_expectations prefect prefect_great_expectations
```

Reply:
Depends on your philosophy. :) I like pointing to versions that I tested worked together. Maybe a moot point with your coming docs testing changes.